src/pygeoprocessing/geoprocessing.py

```diff
@@ -3550,3 +3540,133 @@ def _is_number(x):
             "Invalid value for `target_pixel_size`, expected two numerical "
             "elements, got: %s", repr(target_pixel_size))
     return True
+
+
+def shapely_geometry_to_vector(
+        shapely_geometry_list, target_vector_path, projection_wkt,
+        vector_format, fields=None, attribute_list=None,
+        ogr_geom_type=ogr.wkbPolygon):
+    """Convert list of geometry to vector on disk.
+
+    Args:
+        shapely_geometry_list (list): a list of Shapely objects.
+        target_vector_path (str): path to target vector.
+        projection_wkt (str): WKT for target vector.
+        vector_format (str): GDAL driver name for target vector.
+        fields (dict): a python dictionary mapping string fieldname
+            to OGR Fieldtypes, if None no fields are added
+        attribute_list (list of dicts): a list of python dictionary mapping
+            fieldname to field value for each geometry in
+            `shapely_geometry_list`, if None, no attributes are created.
+        ogr_geom_type (ogr geometry enumerated type): defaults to wkbPolygon.
```

Comment:
I think so but it is not a common case, it's usually when something's broken. Occasionally the geometry type of a layer will not actually be the geometry type of the features, sometimes subtle because of multipolygon in a polyon or such. pygeoprocessing does have the useful concept about exposing the underlying switches if you need them a la' `raster_driver_creation_tuple`.  We often hit those when something's broken and while not as extreme `ogr_geom_type` feels like one of those switches to me. I do agree that docstring is a little light, I explain in b8e4d7dff20fa1629f188b9f2a41b53393f2574a that it's used for setting the target layer's geometry type.

Reply:
Aww crud my comment didn't stick. I've had cases where the underlying polygon types do not correspond to the layer and not have control over it. Say it's a multipolygon but really a polygon? To me this argument feels like the `raster_creation_tuple` option on the other functions in pgp -- it's a switch to something you don't often need but if it's not there when you do need it you have to write your own implementation. I added a better docstring in b8e4d7dff20fa1629f188b9f2a41b53393f2574a to indicate that parameter is for the target layer geometry type

Reply:
The comment stuck!  And thanks for the updated docstring.